src/QMCDrivers/WFOpt/GradientTest.cpp

```diff
@@ -60,20 +59,62 @@ void GradientTest::run(QMCCostFunctionBase& costFunc, const std::string& root_na
     param_deriv_index_++;
   }
 
-  app_log() << "Param_Name  Value    Numeric            Analytic       Percent" << std::endl;
+  size_t max_name_len = std::string("Param_Name").size();
```

Comment:
It is better to define a `std::string_view` for "Param_Name" considering it being used twice here and below.